src/components/list-item/list-item.tsx

```diff
@@ -604,28 +605,35 @@ export class ListItem
   };
 
   toggleSelected = (): void => {
-    if (this.disabled) {
+    const { selectionMode, selected } = this;
+
+    if (this.disabled || selectionMode === "none" || !selectionMode) {
       return;
     }
 
-    if (this.selectionMode === "multiple" || this.selectionMode === "single") {
-      this.selected = !this.selected;
-    } else if (this.selectionMode === "single-persist") {
+    if (selectionMode === "multiple" || selectionMode === "single") {
+      this.selected = !selected;
+    } else if (selectionMode === "single-persist") {
       this.selected = true;
     }
 
     this.calciteListItemSelect.emit();
   };
 
   handleItemKeyDown = (event: KeyboardEvent): void => {
+    if (event.defaultPrevented) {
+      return;
+    }
+
     const { key } = event;
     const composedPath = event.composedPath();
-    const { containerEl, contentEl, actionsStartEl, actionsEndEl, open, openable } = this;
+    const { containerEl, contentEl, actionsStartEl, actionsEndEl, open, openable, selectionMode } =
+      this;
 
     const cells = [actionsStartEl, contentEl, actionsEndEl].filter(Boolean);
     const currentIndex = cells.findIndex((cell) => composedPath.includes(cell));
 
-    if (key === " ") {
+    if (isActivationKey(key) && selectionMode && selectionMode !== "none") {
```

Comment:
`treegrid` role says enter should perform default row action and not space. cc @geospatialem what do you think? Should we just honor `Enter` key like the spec defines? 

https://www.w3.org/WAI/ARIA/apg/patterns/treegrid/examples/treegrid-1/

Reply:
Yes, concur with changing to `Enter` from `space` for alignment, and also seems more intuitive. Once updated, can update the docs site to reflect the change (looks like `space` was not previously doc'd).

Reply:
@geospatialem should the "default action" be to toggle selection or toggle open state of children?

Reply:
Was the children open state previously handled by `enter`? Thought it was through the arrow keys? Would assume the default would be toggle selection.